demo/app.py

```diff
@@ -45,3 +45,33 @@ async def chat(
     return LLMChainStreamingResponse(
         chain, request.query, media_type="text/event-stream"
     )
+
+def retrieval_qa_chain():
+    from langchain.chains import RetrievalQAWithSourcesChain
+    from langchain.chains.qa_with_sources import load_qa_with_sources_chain
+    from langchain.chains.qa_with_sources.stuff_prompt import PROMPT as QA_PROMPT
+    from langchain.vectorstores import FAISS
+    from langchain.embeddings import OpenAIEmbeddings
+
+    callback_manager = AsyncCallbackManager([])
+    vectorstore = FAISS.load_local(index_name="langchain-python", embeddings=OpenAIEmbeddings(), folder_path="demo/")
```

Comment:
i will change this to something simpler like `state_of_union.txt` because its a common example in langchain. not in this PR.